tests/database/crud/test_delete.py

```diff
@@ -0,0 +1,64 @@
+"""Module to test the delete functionality of the genotype API CRUD."""
+
+from genotype_api.database.models import Analysis, Sample, User, Plate, SNP
+from genotype_api.database.store import Store
+
+
+def test_delete_analysis(base_store: Store, test_analysis: Analysis):
+    # GIVEN an analysis and a store with the analysis
+    assert base_store._get_query(Analysis).all()[0]
```

Comment:
I think this is more exact, but not sure how many analyses would be in the store. If there are too many this would take a while
```suggestion
    assert test_analysis in base_store._get_query(Analysis).all()
```